tensorflow_probability/examples/bayesian_neural_network.py

```diff
@@ -190,23 +183,20 @@ class Dummy(object):
   mnist_data = Dummy()
   mnist_data.train = Dummy()
   mnist_data.train.images = np.float32(np.random.randn(
-      num_examples, np.prod(IMAGE_SHAPE)))
+      num_examples, *IMAGE_SHAPE))
```

Comment:
Does this work?

Reply:
yes. It works.

Reply:
Oh, I was just surprised that *IMAGE_SHAPE worked. Until just now I thought it was somehow shorthand for taking the product. I now realize it's passing it into *args. :)